pkg/time/time.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package time
+
+import (
+	"runtime"
+
+	"github.com/pingcap/chaos-mesh/pkg/mapreader"
+	"github.com/pingcap/chaos-mesh/pkg/ptrace"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var log = ctrl.Log.WithName("time")
+
+// TODO: auto generate these codes
+var fakeImage = []byte{
+	0xb8, 0xe4, 0x00, 0x00, 0x00, //mov    $0xe4,%eax
```

Comment:
These codes are "linked" manually. Maybe we can write a simple linker later (which seems little complicate)